CMakeLists.txt

```diff
@@ -141,6 +141,10 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     option(NGRAPH_ONNXIFI_ENABLE "Enable ONNX Interface for Framework Integration" TRUE)
 endif()
 
+if (NGRAPH_UNIT_TEST_ENABLE AND NOT NGRAPH_TOOLS_ENABLE)
```

Comment:
Can we just disable the tool unit test if tools are disabled and tests are enabled? I don't think there should be a cmake error related to config of tools or test.